src/components/footer/footer.js

```diff
@@ -22,35 +23,34 @@ const socialLinks = [
 
 const Footer = () => {
   return (
-    <footer className="custom-footer">
-      <div className="custom-footer__logo">
-        <Logo />
-      </div>
-      <div className="links-to-pages">
-        {links.map(link => (
-          <Link key={link.id} to={link.path} className="nav-link link">
-            <i className="link-circle fas fa-circle link-text" />
-            {link.name}
-          </Link>
-        ))}
+    <footer className="footer__container">
+      <div className="footer__links_and_logo_container">
+        <Link to="/home">
+          <img src={LogoWhite} alt="Logo" className="footer__logo" />
+        </Link>
+        <div className="footer__links_to_pages_container">
+          {links.map(link => (
```

Comment:
It probably missed during latest PRs, but I'd like to ask you to fix it in this PR.
This `map` call makes from links array, which is constant and will not be changed. So all components, which are created from these links are always same.
Could you please extract this `map` call from this place and move it outside of component function into some constant variable

Reply:
DOne